include/clang/CrossTU/CrossTranslationUnit.h

```diff
@@ -85,7 +90,8 @@ class IndexError : public llvm::ErrorInfo<IndexError> {
 /// \return Returns a map where the USR is the key and the filepath is the value
 ///         or an error.
 llvm::Expected<llvm::StringMap<std::string>>
-parseCrossTUIndex(StringRef IndexPath, StringRef CrossTUDir);
+parseCrossTUIndex(StringRef IndexPath, StringRef CrossTUDir,
+                  StringRef CompilationDatabase);
```

Comment:
Perhaps the functions which take `StringRef CompilationDatabase` as a parameter could rather take an `Optional<StringRef>`. That way we could leave the enforced meaning of an empty string as a not-set-value.

Reply:
Fixed.